app/app.yaml

```diff
@@ -19,13 +19,19 @@ api_version: 1
 threadsafe: false
 
 handlers:
+# Remote API handlers. Note that login: admin must not be specified because the
+# handler performs its own authentication. /personfinder/remote_api is defined
+# for google.org/personfinder, which only accepts paths starting with
+# /personfinder.
+- url: /remote_api
```

Comment:
It would work too, but I thought that it may be better to be specific rather than allowing it to match random paths e.g., /japan/view/remote_api or something. But I don't have strong opinion.